appveyor.yml

```diff
@@ -1,8 +1,11 @@
-image: Visual Studio 2017
+image: Visual Studio 2019
 
 init:
   - git config --global core.autocrlf input
 
+install:
+  - choco install dotnetcore-sdk --version 3.0.100
```

Comment:
Can remove this - project is using the (newer) one in the image anyway, so this is just eating time on the build really